UI-1-github_login/github-login.html

```diff
@@ -1,62 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Document</title>
     <style>
-        body{
-            width:250px;
-            font-family:'Trebuchet MS';
-            font-size: 11px;
-            color: grey;
-        }
+      body {
+        width: 250px;

```

Comment:
We never touch the body. Always make a container inside the body if you want to manipulate the page's width